src/style.css

```diff
@@ -1,3 +1,88 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(./font/Roboto-Medium.ttf);
+}
+
+body {
+  font-family: Roboto, Arial, sans-serif;
+  background-color: gray;
+  display: flex;
+  justify-items: center;
+  font-size: 12px;
+}
+
+header {
+  width: 100%;
+  background-color: rgba(255, 255, 255, 1);
+  text-align: center;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo img {
```

Comment:
Check in browser: while your image has correct width-height of 40*40, the .logo container has 40*43 properties.

Reply:
Sprawdziłem na stronie w dev tools i obrazek ma równo 40x40 px.